presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -731,6 +732,66 @@ exists as we've rolled back to the previous state.
     -----------+------+-----------+---------
     (0 rows)
 
+Time Travel using VERSION and TIMESTAMP
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use iceberg connector to access the historical data of a table.
+This means you can see how the table looked like at a certain point in time,
+even if the data has changed or been deleted since then.
+
+.. code-block:: sql
+
+    // snapshot ID 5300424205832769799
+    INSERT INTO ctas_nation VALUES(10, 'united states', 1, 'comment');
+
+    // snapshot ID 6891257133877048303
+    INSERT INTO ctas_nation VALUES(20, 'canada', 2, 'comment');
+
+    // snapshot ID 705548372863208787
+    INSERT INTO ctas_nation VALUES(30, 'mexico', 3, 'comment');
+
+    // snapshot ID for first record
+    SELECT * FROM ctas_nation FOR VERSION AS OF 5300424205832769799;
+
+.. code-block:: text
+
+     nationkey |      name     | regionkey | comment
+    -----------+---------------+-----------+---------
+            10 | united states |         1 | comment
+    (1 row)
+
+You can also access the historical data of a table by providing a time in the past,
```

Comment:
```suggestion
You can access the historical data of a table using ``FOR TIMESTAMP AS OF TIMESTAMP``. 
```
Suggestion to emphasize how the reader can access historical data. Is this correct?